src/backend/app/projects/project_crud.py

```diff
@@ -2359,14 +2359,12 @@ async def get_pagination(page: int, count: int, results_per_page: int, total: in
     return pagination
 
 
-async def get_dashboard_detail(project_id: int, db: Session):
+async def get_dashboard_detail(project: db_models.DbProject, db: Session):
     """Get project details for project dashboard."""
-    project = await get_project(db, project_id)
-    db_organisation = await organisation_crud.get_organisation_by_id(
+    db_organisation = await organisation_deps.get_organisation_by_id(
```

Comment:
You can see elsewhere in the code how to use Depends.

It should be in the function params:

```python
endpoint_function(org: db_models.DbOrganisation = Depends(organisation_deps.get_organisation_by_id), db: ...):
```

Then you will see the organisation id query param inherited from the dependency.

Reply:
how is org_id inherited to the function `get_organisation_id` ? since no param `org_id` is passed in the path?

Reply:
https://docs.hotosm.org/dev-guide/web-apis/#dependencies-can-include-route-parameters

As org_id is a param in the dependency: https://github.com/hotosm/fmtm/blob/6b3214207ce8e5783fac7740faaaa57041af47c8/src/backend/app/organisations/organisation_deps.py#L51

Then org_id will be passed as a required route parameter to the endpoint that uses `Depends`.